pysat/__init__.py

```diff
@@ -100,13 +100,14 @@
 with Lock(version_filename, 'r', params['file_timeout']) as version_file:
     __version__ = version_file.read().strip()
 
-from pysat import utils
-from pysat._instrument import Instrument
-from pysat._meta import Meta, MetaLabels
+from pysat._constellation import Constellation
```

Comment:
Constellation previously had to be imported after everything else.  Might not work on all systems to have this in alphabetical order.

Reply:
OK.  Putting a blank line in between import blocks would reset the alphabetical check.